dotcom-rendering/src/web/components/TopicFilterBank.importable.tsx

```diff
@@ -83,6 +83,9 @@ export const TopicFilterBank = ({
 	filterKeyEvents = false,
 	id,
 }: Props) => {
+	const showFilters = keyEvents?.length && availableTopics.length;
```

Comment:
My brain melts in these situations so there's a non trivial chance I'm talking rubbish but should this be OR here or?

```suggestion
	const showFilters = keyEvents?.length || availableTopics.length;
```

Reply:
Oops yeah, you're absolutely right!